recipes/sqlite/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version="3.11.1" %}
+{% set version_split=version.split(".") %}
+{% set version_coded="".join([version_split[0],
+                              str(int(version_split[1])*100),
+                              str(int(version_split[2])*100)
+                              ]) %}
```

Comment:
I think the linter is struggling with this. It is the only thing that seems complex enough to cause it issues and jinja is its weak point. Not sure what (if anything) we should do about it.

Raising to your attention, @pelson.